src/main/java/org/biscuitsec/biscuit/datalog/SymbolTable.java

```diff
@@ -262,6 +262,12 @@ public SymbolTable(SymbolTable s) {
         publicKeys.addAll(s.publicKeys);
     }
 
+    public SymbolTable(List<String> symbols) {
+        this.symbols = new ArrayList<>();
```

Comment:
you can init directly the list:
```suggestion
        this.symbols = new ArrayList<>(symbols);
```